exercises/jigsaw-puzzle/metadata.toml

```
@@ -0,0 +1,3 @@
+title = "Jigsaw Puzzle"
+blurb = "Fill in missing jigsaw puzzle details from partial data"
+source = "atk just started another 1000-pieces jigsaw puzzle when this idea hit him"
```

Could you add the source URL?

```suggestion
source = "atk just started another 1000-pieces jigsaw puzzle when this idea hit him"
source_url = "https://github.com/exercism/problem-specifications/pull/2554"
```